docs/installation/data-preparation-scripts.rst

```diff
@@ -1,4 +1,4 @@
-Data Preperation Scripts
+Data Preparation Scripts
```

Comment:
Ha!  Nice catch.

Reply:
Pin added 'preperation' to the wordlist when pyspell caught the typo...